Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tauri-plugin-localhost"
-version = "0.1.0"
+version = "0.1.1"
```

Comment:
Please revert this and add a change file instead: https://github.com/tauri-apps/tauri-plugin-localhost/tree/dev/.changes

Reply:
> Please revert this and add a change file instead: https://github.com/tauri-apps/tauri-plugin-localhost/tree/dev/.changes

done.